build.gradle

```diff
@@ -331,6 +335,7 @@ allprojects {
 
                     // Force consistency for dependencies from cloud
                     force "joda-time:joda-time:${jodaTimeVersion}"
+                    force "joda-time:joda-time:${jodaTimeVersion}"
```

Comment:
Oops

Reply:
Fixed. I was hacking at it in the hopes of eliminating the force, but it's still needed (only one though!)